src/plugin/asyncMessageHandlers/getThemeReferences.ts

```diff
@@ -16,13 +16,13 @@ export async function getThemeReferences(prefixStylesWithThemeName?: boolean) {
   const activeThemes = themeInfo.themes?.filter((theme) => Object.values(themeInfo.activeTheme).some((v) => v === theme.id));
   const stylePathPrefix = prefixStylesWithThemeName && activeThemes.length > 0 ? activeThemes[0].name : undefined;
 
-  activeThemes?.forEach((theme) => {
-    Object.entries(theme.$figmaVariableReferences ?? {}).forEach(async ([token, variableId]) => {
+  activeThemes?.forEach(async (theme) => {
```

Comment:
If we don't await it at the top level it won't block the execution of this function till after all have been completed

```suggestion
  await Promise.all(activeThemes?.map(async (theme) => {
```